include/boost/graph/bc_clustering.hpp

```diff
@@ -131,9 +128,8 @@ void betweenness_centrality_clustering(MutableGraph& g, Done done,
         brandes_betweenness_centrality(g,
             edge_centrality_map(edge_centrality)
                 .vertex_index_map(vertex_index));
-        std::pair< edge_iterator, edge_iterator > edges_iters = edges(g);
-        edge_descriptor e
-            = *boost::first_max_element(edges_iters.first, edges_iters.second, cmp);
+        auto edges_iters = edges(g);
+        auto e = *max_element(edges_iters.first, edges_iters.second, cmp);
```

Comment:
I haven't checked the whole diff, but this one change that doesn't fit with the stated purpose of the pull request is broken.  :)

Reply:
Thanks. Resolving the conflicts was awkward. I'll mark this as Ready for Review again when I've checked more carefully.